corehq/apps/app_manager/templates/app_manager/v1/partials/case_list.html

```diff
@@ -116,6 +116,16 @@
 
 {% include 'app_manager/v1/partials/case_list_filtering.html' %}
 
+{{ request|toggle_tag_info:"DETAIL_LIST_TAB_NODESETS" }}
+{% if request|toggle_enabled:'DETAIL_LIST_TAB_NODESETS' %}
+<div data-bind="with: shortScreen">
+    <input data-bind="checked: sortNodesetColumns" type="checkbox">
+    <span>
+        {% trans "Sort entries in the Nodeset tabs" %}
```

Comment:
Minor, but it'd be a ux benefit to make this a label so users can click on the text to check the checkbox. [This](http://getbootstrap.com/css/#forms-controls) has the typical markup for a label+checkbox in bootstrap.